src/applications/vaos/components/VideoLink.unit.spec.jsx

```diff
@@ -158,14 +174,18 @@ describe('VAOS Component: VideoLink', () => {
   });
 
   it('renders join appointment link within 4 hours after start time', () => {
-    const now = moment();
+    const now = new Date();
     const store = createTestStore(initialState);
     const appointment = {
       location: {
         vistaId: '983',
         locationId: '983',
       },
-      start: moment.tz(now, 'America/Los_Angeles').add(90, 'minutes'),
+      start: formatInTimeZone(
+        addMinutes(now, 90),
```

Comment:
I'm surprised that any of these tests work. There's a few issues here:
1. the Facility is Cheyenne which is in the MT timezone but the start times are in PT/ET. Though technically not wrong, it makes things quite difficult to track.
2. I suspect the addMinutes/subMinutes are reversed. The component logic says "Button is enabled 30 minutes prior to start time, until 4 hours after start time" which means start should be between `subMinutes(now, 240)` to `addMinutes(now, 30)` for the button to show up.
3. All this might be irrelevant since this logic is moving to the backend. The new logic is tested below in the `'with vaOnlineSchedulingFeSourceOfTruthTelehealth=true'` section